src/pages/profile/basic/model.ts

```diff
@@ -0,0 +1,54 @@
+import { AnyAction, Reducer } from 'redux';
+
+import { EffectsCommandMap } from 'dva';
+import { BasicGood } from './data.d';
+import { queryBasicProfile } from './service';
+
+export interface StateType {
+  basicGoods: BasicGood[];
+}
+
+export type Effect = (
+  action: AnyAction,
+  effects: EffectsCommandMap & { select: <T>(func: (state: StateType) => T) => T },
+) => void;
+
+export interface ModelType {
+  namespace: string;
+  state: StateType;
+  effects: {
+    fetchBasic: Effect;
+  };
+  reducers: {
+    show: Reducer<StateType>;
+  };
+}
+
+const Model: ModelType = {
+  namespace: 'profileAndbasic',
+
+  state: {
+    basicGoods: [],
+  },
+
+  effects: {
+    *fetchBasic(_, { call, put }) {
+      const response = yield call(queryBasicProfile);
+      yield put({
+        type: 'show',
+        payload: response,
+      });
+    },
+  },
+
+  reducers: {
+    show(state, { payload }) {
+      return {
+        ...state,
+        ...payload,
+      };
+    },
+  },
+};
+
+export default Model;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of default exports is forbidden](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=4583897)